internal/scanner/scanner.go

```diff
@@ -422,8 +432,8 @@ func (s *Scanner) scanURL(ctx context.Context, w *testWork) error {
 		}
 
 		newCtx := ctx
-		if w.testHeaderValue != "" {
-			newCtx = metadata.AppendToOutgoingContext(ctx, "X-GoTestWAF-Test", w.testHeaderValue)
+		if w.debugHeader != "" {
+			newCtx = metadata.AppendToOutgoingContext(ctx, "X-GoTestWAF", w.debugHeader)
```

Comment:
`X-GoTestWAF` value should be definitely a constant as I see, since you use it basically everywhere here

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/160/commits/a2fa101fb98a085827aeabdfd6d7ccd7f91ca74b